tests/coverage/RemoteWithCoberturaCept.php

```diff
@@ -0,0 +1,8 @@
+<?php
+$I = new CoverGuy($scenario);
+$I->wantTo('try generate remote cobertura xml report');
```

Comment:
This test fails with PHPUnit 8 and older so it must be skipped if `\SebastianBergmann\CodeCoverage\Report\Cobertura` class does not exist.

Please check if this code works:
```php
if (!class_exists(\SebastianBergmann\CodeCoverage\Report\Cobertura::class)) {
  $scenario->skip('Cobertura report requires php-code-coverage 9.2');
}
````